mitodl/odl-video-service
# Add Video.custom_order field

#### What are the relevant tickets?
Closes #726 

#### What's this PR do?
- Adds `custom_order` to `Video` model
- Changes sorting order to `['custom_order', '-created_at']`
- Reorders some imports that pylint suddenly started complaining about

#### How should this be manually tested?
- Upload some videos to a collection or look at an existing collection with multiple videos.  The videos should be displayed in reverse chronological order (newest first).
- Go to the admin page and for at least some of those videos, enter an integer for `custom_order`.
- Go back to the collection detail page.  Videos with a non-null `custom_order` should be displayed first, from lowest to highest `custom_order` value, followed by videos without a `custom_order`, in reverse chronological order.  If two videos have the same `custom_order` value, they should be sorted in reverse chronological order relative to each other.
